tools.sh

```diff
@@ -207,7 +212,7 @@ function create_vm {
         --cdrom /var/lib/libvirt/${SNC_PRODUCT_NAME}/${iso} \
         --events on_reboot=restart \
         --autoconsole none \
-        --boot uefi \
+        ${bootOption} \
```

Comment:
i think in crc also we create the VM and uefi boot loader, at least for macOS, will the bundle work in that case?

Reply:
@anjannath OKD doesn't have arm64 support and iirc it was required for that.